src/plugins/intel_gpu/src/graph/eltwise.cpp

```diff
@@ -408,6 +423,14 @@ void eltwise_inst::check_inputs_count(eltwise_node const& node) {
                     node.id(),
                     "Invalid eltwise inputs number (should be equal to 2). Actual: " + std::to_string(inputs_number));
             break;
+        case eltwise_mode::is_finite:
+        case eltwise_mode::is_inf:
+        case eltwise_mode::is_nan:
+            if (inputs_number != 1)
+                CLDNN_ERROR_MESSAGE(
```

Comment:
Please use OPENVINO_ASSERT.

Reply:
Used as above to be consistent. But if needed, I can change both places to use OPENVINO_ASSERT.

Reply:
Yes those are remaining legacy and we are replacing with OPENVINO_ASSERT for newly added codes.

Reply:
Done